chart/kubeapps/templates/apprepositories.yaml

```diff
@@ -28,6 +28,14 @@ spec:
   {{- if $.Values.securityContext.enabled }}
   syncJobPodTemplate:
     spec:
+      containers:
```

Comment:
(see my comment below)

to avoid setting this variables unnecessarily I would check if this is enabled: `{{- if $.Values.initialRepos.proxy.enabled }}`